src/sentry/monitors/logic/mark_failed.py

```diff
@@ -189,9 +189,11 @@ def mark_failed_threshold(failed_checkin: MonitorCheckIn, failure_issue_threshol
     if monitor_muted:
         return True
 
-    for previous_checkin in previous_checkins:
-        checkin_from_db = MonitorCheckIn.objects.get(id=previous_checkin["id"])
-        create_issue_platform_occurrence(checkin_from_db, fingerprint)
+    # Do not create event/occurrence if we don't have a fingerprint
+    if fingerprint:
+        for previous_checkin in previous_checkins:
+            checkin_from_db = MonitorCheckIn.objects.get(id=previous_checkin["id"])
```

Comment:
I guess it's not a huge issue since the threshold is low, but we really should fetch these using `.filter(id__in=...)`

Reply:
Will clean this up 👍 agreed

Reply:
https://github.com/getsentry/sentry/pull/66327